src/store/auth.ts

```diff
@@ -31,9 +34,26 @@ export const useAuthStore = defineStore('authStore', {
       return baseURL.startsWith('http') ? baseURL : `https://${baseURL}.hotwax.io/api/`
     },
     getRedirectUrl: (state) => state.redirectUrl,
+
+    isValidOMS(oms)=>{
+      if(oms === "dev-apps"){
```

Comment:
Here we have hardcoded the oms value, but then this condition will always return false if we have used any other oms which is not correct.
Need to have some robust solution that can handle all the oms and we also don't need to hardcode oms names for checking.